release/build.gradle.kts

```diff
@@ -30,6 +30,7 @@ val library = project.extensions.extraProperties["library"] as Map<String, Map<S
 dependencies {
   compile(library.getValue("groovy").getValue("groovy_all"))
   compile("commons-cli:commons-cli:1.2")
+  permitUnusedDeclared("commons-cli:commons-cli:1.2") // BEAM-11761
```

Comment:
@ibzib do you think this is necessary for the "release" project?

Reply:
Not sure, but permitUnusedDeclared is fine either way.